### ISSUE TITLE
In AAE or VAAE the EarlyStopping Callback only accepts monitor with name "loss"

### ISSUE THREAD

**Issue by RovshanShirinli** [kkahloots/Generative_Models#2]
`es = tf.keras.callbacks.EarlyStopping(
    monitor='loss',  #should be 'val_loss'
    min_delta=1e-12, 
    patience=5, 
    verbose=1, 
    restore_best_weights=True
)`
Suspected cause: the loss and metrics names list does not have "val_loss"

**Comment by kkahloots** [comment]
is it a warning or an error?
if warning, ignore it as long as you pass a validation_data parameter

**Comment by RovshanShirinli** [comment]
It was a warning but when i tried to replicate the error right now it did not show up with verbosity=0 (before it was 0 as well). I just re-tried Pacman_AAE_Dense
